sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/queries/SelectQueryTypeTest.kt

```diff
@@ -336,7 +340,7 @@ class SelectQueryTypeTest {
       |FROM data
       |WHERE token = :token
       |  AND id IN ?
-      |  AND (token != :token OR (name = :name OR :name IS NULL))
+      |  AND (token != :token OR (name = :name OR :name = 'foo'))
```

Comment:
This is just updating an existing test that was broken under this change, so I moved it to something more readable.

Reply:
coolcool